src/motion-queue.h

```diff
@@ -72,9 +75,27 @@ struct MotionSegment {
 #endif
 } __attribute__((packed));
 
+// Layout of the status register
```

Comment:
Maybe mention why you do the 24/8 bit split: assuming atomicity of 32 bit boundaries.